docs/project-requirements.md

```diff
@@ -236,32 +236,38 @@ W każdym story pomijamy aktora, jest napisane w nagłówku kto nim jest. Dodatk
 # Kryteria akceptacji projektu dla I semestru prac
 
 ## Wymagane
-- Wybór gracza do pojedynku (US. 1.2.2)
-- Pojedynek między dwoma graczami (US. 1.1.8, 1.1.9)
-- Talia Kart (US. 1.1.13, 1.4.2 - 1.4.7, 1.1.15)
-- Profil Użytkownika (US. 1.4.1, 1.4.2, 1.1.5, 1.1.6, 1.1.14)
-- Edytor Kart (US. 2.2.1 - 2.2.9, 2.4.4)
 - 20 Kart stworzonych w edytorze (US. 1.1.19)
-- Testy użyteczności trybu Battle i Edytora Kart
-- Serwer produkcyjny dla grupy odbiorców testującej produkt
+- Edytor Kart (US. 2.2.1 - 2.2.9, 2.4.4)
 
 ## Oczekiwane
-- System rankingowy Battle (US. 1.1.10, 1.1.12, 1.1.18)
-- Balans kart (US. 1.1.12, 1.1.17)
-- Tabele rankingowe (US. 1.3.1 - 1.3.7)
+- Pojedynek między dwoma graczami - symulator
+- Serwer produkcyjny dla grupy odbiorców testującej produkt

```

Comment:
Co powiecie na to, żeby przenieść Serwer produkcyjny do wymaganych?

Reply:
imo bardzo w porządku patrząc na charakter apki